### PR TITLE
feat(integration): add Bland.ai

### PR DESCRIPTION
**Pull request by jwd-dev**
## Describe your changes
Need this for a beta release of respell. Docs are here: docs.bland.ai

### CODE REVIEW

**Review by bodinsamuel — COMMENTED** [review]
Thanks for your PR 🙏🏻 

I'll suggest to rename to blandai since it's a very common name, to avoid any future conflict.
There are few missing file update, I'll suggest to take a look at previous one to see what's missing: https://github.com/NangoHQ/nango/pull/2102/files

We are curious about your use case, out of curiosity can you tell us more? Privately in slack if you prefer ☺️

**Comment by jwd-dev** [comment]
Updated.

File: packages/webapp/public/images/template-logos/blandai.svg
Comment (bodinsamuel) [inline packages/webapp/public/images/template-logos/blandai.svg:1]: @hassan254-prog any idea how to get a better svg? I have put a png inside a svg, not ideal maybe you have a trick :D

**Review by khaliqgant — APPROVED** [review]

File: packages/webapp/public/images/template-logos/blandai.svg
Comment (TBonnin) [reply packages/webapp/public/images/template-logos/blandai.svg:1]: I am gonna merge this PR. However @hassan254-prog I am interested to know your answer to this question. Feel free to open a PR if you can improve the svg

**Review by TBonnin — APPROVED** [review]

**Pull request merged**